build/commands/lib/start.js

```diff
@@ -84,6 +84,15 @@ const start = (passthroughArgs, buildConfig = config.defaultBuildConfig, options
     killSignal: 'SIGTERM'
   }
 
+  if (process.platform === 'darwin' && !config.isReleaseBuild()) {
+    // Disable 'accept incoming network connections' and 'keychain access'
+    // dialogs in MacOS. See //docs/mac_build_instructions.md for details.
+    braveArgs.push('--use-mock-keychain')
```

Comment:
let's also add `--use-real-keychain` to `npm run start`.

Reply:
done